disnake/interactions/base.py

```diff
@@ -367,7 +371,7 @@ async def original_message(self) -> InteractionMessage:
         self._original_message = message
```

Comment:
internal field should be renamed as well

Reply:
https://github.com/DisnakeDev/disnake/pull/738/commits/7472b179471d9f61eed4c9341ca6d31d25d618a9